src/test/java/api/support/APITests.java

```diff
@@ -251,6 +254,7 @@ public void beforeEach() throws InterruptedException {
     itemsFixture.cleanUp();
 
     usersClient.deleteAllIndividually();
+//    templateClient.deleteAll();
```

Comment:
Why do you need this comment? Personally, I'd like to remove it.

Reply:
It's been commented by mistake, fixed